sources/niLang/src/API/niLang/Utils/CrashReport.h

```diff
@@ -4,6 +4,10 @@
 // SPDX-License-Identifier: MIT
 #include "../Types.h"
 #include "../StringDef.h"
+#ifndef niNoExceptions
```

Comment:
I am not sure if I did the right thing. How is it possible that this was compiling in other machines (maybe wasn't?)
Anyways, if we need to call set_terminate bellow we need to include this somewhere. If this is not the right place let me know.

Reply:
That's correct I think. Some of the libraries wont have exceptions enabled and that should be setup correctly for them indeed. That might be an issue on MSVC only though, Clang/GCC can still include the exception headers with exceptions disabled I think.